client/keys/list_test.go

```diff
@@ -36,7 +37,7 @@ func Test_runListCmd(t *testing.T) {
 		mockIn.Reset("testpass1\ntestpass1\n")
 	}
 
-	_, err = kb.CreateAccount("something", tests.TestMnemonic, "", "", 0, 0)
+	_, err = kb.CreateAccount("something", tests.TestMnemonic, "", "", "", keys.Secp256k1)
```

Comment:
why an empty hdpath assume it uses the 0 account and index values?

Reply:
this should actually fail as the hdPath provided is invalid. Please add a testcase for that

Reply:
Empty hdpath doesn't assume 0 account and index values, its an empty hdPath.  This is valid according to Bip32, it's just saying don't use HD derivation.

This was purposely generalized to allow any valid bip32 path, not just bip44 compliant.  By default, the CLI still uses BIP44 as to not have API breakage.

Reply:
The purpose of generalizing this was to allow some signing algos to use their own HD derivation methodology (sr25519 does this).  CreateAccount` doesn't have any restriction now on what HD Path strings need to look like.